webview/src/java/com/defold/webview/WebViewJNI.java

```diff
@@ -309,6 +247,14 @@ private WebViewInfo createView(Activity activity, int webview_id)
         info.windowParams.width = WindowManager.LayoutParams.MATCH_PARENT;
         info.windowParams.height = WindowManager.LayoutParams.MATCH_PARENT;
         info.windowParams.flags = WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL;
+        info.windowParams.softInputMode = WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING;
+        if (Build.VERSION.SDK_INT < 30) {
+            info.windowParams.systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN | View.SYSTEM_UI_FLAG_LAYOUT_STABLE
+                        | View.SYSTEM_UI_FLAG_FULLSCREEN | View.SYSTEM_UI_FLAG_HIDE_NAVIGATION | View.SYSTEM_UI_FLAG_IMMERSIVE;
+        }
+        if (Build.VERSION.SDK_INT >= 28) {
+            info.windowParams.layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS;
+        }
```

Comment:
How does this work? What happens if Build.VERSION.SDK_INT is >= 30?

Does `LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS` replace all of the other flags from API 28 and upwards?

Do we need to run both of those conditional checks on API level 28 and 29? Or is it enough to set `LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS`?

Reply:
Ah, I just checked again and see that although `layoutInDisplayCutoutMode` available since API level 28 but `LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS` is added since API level 30, so it should be `Build.VERSION.SDK_INT >= 30`.
Using `LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS` will allow the webview display behind status bar & cutout area (it means the webview origin will be placed at them same game window origin, useful for calculating & displaying it at the same place with GUI).
I've tried some other flags but this flag works on my device, so I picked it.

Reply:
I ended up with `LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES` which DefoldActivity using